pkg/raft/raft.go

```diff
@@ -250,18 +250,15 @@ type Config struct {
 	// See: https://github.com/etcd-io/raft/issues/80
 	DisableConfChangeValidation bool
 
-	// TestingDisablePreCampaignStoreLivenessCheck may be used by tests to disable
-	// the check performed by a peer before campaigning to ensure it has
-	// StoreLiveness support from a majority quorum.
-	TestingDisablePreCampaignStoreLivenessCheck bool
-
 	// StoreLiveness is a reference to the store liveness fabric.
 	StoreLiveness raftstoreliveness.StoreLiveness
 
 	// CRDBVersion exposes the active version to Raft. This helps version-gating
 	// features.
 	CRDBVersion clusterversion.Handle
-	Metrics     *Metrics
+
+	Metrics      *Metrics
+	TestingKnobs *RaftTestingKnobs
```

Comment:
nit: since we're in package `raft`, the name can be shortened to `raft.TestingKnobs`.